examples/kepler-integration/src/components/export-video/export-video-panel-container.js

```diff
@@ -90,19 +70,35 @@ export class ExportVideoPanelContainer extends Component {
       mediaType: 'GIF',
       cameraPreset: 'None',
       fileName: 'Video Name',
-      //  quality: "High (720p)",
+      quality: 'High (720p)',
       viewState: this.props.mapData.mapState,
+      currentWidth: 1280,
+      currentHeight: 720,
       durationMs: 1000,
       encoderSettings: {
         framerate: 30,
         webm: {
+          // TODO width/height not in WEBMEncoder
```

Comment:
This is fine. They don't need to do rescaling.

Reply:
does that apply to the JPEGEncoder as well?

Reply:
Yeah. Gif is the only one that has it because it's unrealistic to export large res gifs